contracts/base/Dispatcher.sol

```diff
@@ -79,24 +84,32 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, Callbacks
                             address recipient,
                             uint256 amountIn,
                             uint256 amountOutMin,
-                            address[] memory path,
+                            /* address[] memory path */
+                            ,
                             bool payerIsUser
                         ) = abi.decode(inputs, (address, uint256, uint256, address[], bool));
+                        address[] calldata path = inputs.toAddressArray(3);
                         address payer = payerIsUser ? msg.sender : address(this);
                         v2SwapExactInput(recipient.map(), amountIn, amountOutMin, path, payer);
                     } else if (command == Commands.V2_SWAP_EXACT_OUT) {
                         (
                             address recipient,
                             uint256 amountOut,
                             uint256 amountInMax,
-                            address[] memory path,
+                            /* address[] memory path */
```

Comment:
delete?

Reply:
i think we should keep it for readablility because theres 1 element not being decoded. But going to change the comment formatting so its clearer